### PR TITLE
Upgrade to eslint 2.7

### PR DESCRIPTION
**Pull request by rylanc**

### CODE REVIEW

File: react.js [hunk 1]
```diff
@@ -28,29 +71,25 @@ module.exports = {
...
+    // Prevent usage of unwrapped JSX strings
+    'react/jsx-no-literals': 0,
     // Disallow undeclared variables in JSX
     'react/jsx-no-undef': 2,
     // Enforce PascalCase for user-defined JSX components
     'react/jsx-pascal-case': 2,
+    // Enforce props alphabetical sorting
+    'react/jsx-sort-props': 0,
+    // Enforce spaces before the closing bracket of self-closing JSX elements
+    'react/jsx-space-before-closing': [2, 'always'],
```
Comment (robbiegill) [inline react.js]: I like this style, but there are going to be a lot of violations in existing code - do you plan to fix them all?

**Comment by robbiegill** [comment]
for reference: http://eslint.org/docs/user-guide/migrating-to-2.0.0

**Comment by robbiegill** [comment]
@zp-alfred accept!

**Comment by zp-alfred** [comment]
Accepted by robbiegill!

File: react.js (hunk 1 above)
Comment (rylanc) [inline react.js]: Yeah, it was easy because the rule can be auto-fixed

**Pull request merged**